xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://json.schemastore.org/prettierrc",
+  "semi": false,
```

Comment:
related to this I guess the fact you removed some semi column? Was discussed somewhere?

Reply:
No, it's currently following the default formatting configured in my IDE.
But a lint check is executed during the maven build.

Reply:
Meaning that those file are currently unused, I guess I can remove them.